checkbox-ng/pyproject.toml

```diff
@@ -0,0 +1,38 @@
+[build-system]
+  requires = ["setuptools>=42", "setuptools_scm[toml]>=3.4"]
+  build-backend = "setuptools.build_meta"
+[project]
+  name = "checkbox-ng"
+  #url="https://launchpad.net/checkbox-ng/"
+  authors = [
+      {name = "Zygmunt Krynicki", email = "zygmunt.krynicki@canonical.com"}
```

Comment:
Please enter a name and email for someone who if actually emailed doesn't bounce.